articles/tables.md

```diff
@@ -20,727 +20,191 @@ category: howto
-{% endhighlight %}
-{% endtab %}
+After clicking on "Next", you will see the option to rename your Table and, optionally, define your schema more specifically. 
 
-{% tab R %}
-{% highlight r %}
-# Utilities function is not available in synapser 0.1.25
-cols <- list(
-    Column(name='artist', columnType='STRING', maximumSize=20),
-    Column(name='album', columnType='STRING', maximumSize=20),
-    Column(name='year', columnType='INTEGER'),
-    Column(name='catalog', columnType='STRING', maximumSize=20))
+## Defining a Schema
+Synapse `Tables` require that you specify the types of data in each column included in the `Table`. If you are creating an empty Table, these columns can be specified manually by clicking on **Add Column** and selecting the column type you wish to add. If you are uploading an existing file, Synapse will recommend column types during upload. If allowing Synapse to recommend column types, be sure to verify that these column types represent your data well. You will be presented with Synapse's best guess under the button labeled "Schema Options" during import. 
 
-project <- synGet('syn1901847')
-schema <- Schema(name='Jazz Albums', columns=cols, parent=project)
-{% endhighlight %}
-{% endtab %}
+Synapse currently supports several kinds of text columsn (STRING, LARGE TEXT), dates (as TIMESTAMP), and a variety of numericcolumns (INTEGER, DOUBLE), and including various Synapse IDs directly (ENTITYID, USER). For a complete list of column types and definitions, please see the [REST docs on columnTypes](https://docs.synapse.org/rest/org/sagebionetworks/repo/model/table/ColumnType.html). 
 
-{% tab Web %}
-If you upload a file, the Web interface will automatically detect the table schema. After a few prompts, you will arrive at the option to name your table. You can adjust the schema(e.g. Column Name, Column Type, etc) here by clicking on the **Schema Options** button underneath the name.
-<br>
 <img id="image" src="/assets/images/tableSchema.png">
-{% endtab %}
-
-{% endtabs %}
 
-<br>
+Note: Each analytic client has a utility function to create columns from a data frame. Please see the [Python Docs](https://python-docs.synapse.org/build/html/Table.html) or [R Docs](https://r-docs.synapse.org/articles/tables.html) for examples.
 
-The Table() function takes two arguments, a schema object and data in some form, which can be:
+## Adding Data to Tables
 
-- a path to a CSV file
-- a dataframe
-- a [`RowSet` object](http://docs.synapse.org/rest/org/sagebionetworks/repo/model/table/RowSet.html){:target="_blank"}
-- a list of lists where each of the inner lists is a row
+If you have imported an existing .CSV or .TSV, all of your data will be imported at once. For very large files, it may take a moment for the Table to fully build and index in order to be queried. You can navigate away from the Tables once you have clicked "Create" and you will not lose any data. 
```

Comment:
```suggestion
If you have imported an existing `.csv` or `.tsv`, all of your data will be imported at once. For very large files, it may take a while for the Table to be built and indexed completely before it can be used. You can navigate away from the Tables once you have clicked "Create" and you will not lose any data. 
```